.github/workflows/scripts/test-build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+assert() {
+    E_PARAM_ERR=98
+    E_ASSERT_FAILED=99
+
+    echo "Asserting: $1"
+    echo "Assertion: $2"
+
+    if [ -z "$2" ]; then
+        return $E_PARAM_ERR
+    fi
+
+    message=$1
```

Comment:
is it intentional to declare them after an not at the beginning ?

Reply:
No its a mistake